test/functional/ui/float_spec.lua

```diff
@@ -2033,6 +2066,368 @@ describe('float window', function()
       end
     end)
 
+    it('border with footer', function()
+      local buf = meths.create_buf(false, false)
+      meths.buf_set_lines(buf, 0, -1, true, {' halloj! ',
+                                             ' BORDAA  '})
+      local win = meths.open_win(buf, false, {
+        relative='editor', width=9, height=2, row=2, col=5, border="double",
+        footer = "Left",footer_pos = "left",
+      })
+
+      if multigrid then
+        screen:expect{grid=[[
+        ## grid 1
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [3:----------------------------------------]|
+        ## grid 2
+          ^                                        |
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+        ## grid 3
+                                                  |
+        ## grid 5
```

Comment:
note: as #24824 was merged, some adjustments are needed here. They should be quite regular tho. In these added tests `grid 5` should be changed into `grid 4`, the array index `[5]` into `[4]` and finally `id = 1002` into `id = 1001` .

Reply:
Thanks for the heads up. Done.